.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       run: dotnet build --no-restore --configuration=Release /p:Version=${{steps.gitversion.outputs.semVer}}
 
     - name: Test
-      run: dotnet test --no-build --configuration=Release --verbosity normal
```

Comment:
I added a new functional test project, but with no physical ollama instance for github tests, I needed to explicitly ignore them.